editor.planx.uk/src/@planx/components/Result/Editor.tsx

```diff
@@ -79,106 +83,110 @@ const FlagEditor: React.FC<{
 type Props = EditorProps<TYPES.Result, Result>;
 
 const ResultComponent: React.FC<Props> = (props) => {
-  const formik = useFormik<Result>({
-    initialValues: {
-      flagSet: props.node?.data?.flagSet || Object.keys(flags)[0],
-      overrides: props.node?.data?.overrides || {},
-    },
-    onSubmit: (newValues) => {
-      if (props.handleSubmit) {
-        props.handleSubmit({ type: TYPES.Result, data: newValues });
-      }
-    },
-    validate: () => {},
-  });
-
-  const allFlagsForSet: FlagWithValue[] = [
-    ...(flags[formik.values.flagSet] ?? []),
-    getNoResultFlag(formik.values.flagSet) as FlagWithValue,
+  const allFlagsForSet = (flagSet: FlagSet): FlagWithValue[] => [
+    ...(flags[flagSet] ?? []),
+    getNoResultFlag(flagSet) as FlagWithValue,
   ];
 
   return (
-    <form onSubmit={formik.handleSubmit} id="modal">
-      <TemplatedNodeInstructions
-        isTemplatedNode={formik.values.isTemplatedNode}
-        templatedNodeInstructions={formik.values.templatedNodeInstructions}
-        areTemplatedNodeInstructionsRequired={
-          formik.values.areTemplatedNodeInstructionsRequired
+    <Formik<Result>
+      initialValues={{
+        flagSet: props.node?.data?.flagSet || Object.keys(flags)[0],
+        overrides: props.node?.data?.overrides || {},
+      }}
+      onSubmit={(newValues) => {
+        if (props.handleSubmit) {
+          props.handleSubmit({ type: TYPES.Result, data: newValues });
         }
-      />
-      <ModalSection>
-        <ModalSectionContent title="Result" Icon={ICONS[TYPES.Result]}>
-          <InputRow>
-            <Typography variant="h5" component="h6">
-              <label htmlFor="result-flagSet">Flag set</label>
-            </Typography>
-            <select
-              id="result-flagSet"
-              name="flagSet"
-              value={formik.values.flagSet}
-              onChange={formik.handleChange}
-              required
-              disabled={props.disabled}
-            >
-              {Object.keys(flags).map((flagSet) => (
-                <option key={flagSet} value={flagSet}>
-                  {flagSet}
-                </option>
-              ))}
-            </select>
-          </InputRow>
+      }}
+      validationSchema={validationSchema}
+      validateOnBlur={false}
+      validateOnChange={false}
+    >
```

Comment:
Replacing the `useFormik()` hook with the `<Formik>` context component allows us to call `useFormikContext<Result>` in the child (without prop drilling the errors).

I've avoided other components that use the `ListManager` so far - I think this is the pattern I'll need to repeat there.